package.json

```diff
@@ -21,14 +21,15 @@
   },
   "dependencies": {
     "atom-package-deps": "^4.0.1",
-    "load-plugin": "^1.0.0",
-    "remark": "4.2.2",
-    "remark-lint": "4.0.2"
+    "remark": "5.0.1",
+    "remark-lint": "4.0.2",
+    "unified-engine-atom": "^1.0.0"
   },
   "devDependencies": {
-    "eslint": "^2.9.0",
-    "eslint-config-airbnb-base": "^3.0.1",
-    "eslint-plugin-import": "^1.7.0"
+    "eslint": "^3.1.1",
+    "eslint-config-airbnb-base": "^5.0.0",
+    "eslint-plugin-import": "^1.7.0",
```

Comment:
This should be `^1.10.2` at a minimum (that was the first working version that supported `eslint@3`.